### PR TITLE
[Gallery] Update desktop Rally for pages that did not have designs before

### PR DESCRIPTION
**Pull request by perclasson**
Changes
===
These screens lacked designs for desktop previously. I asked the designer for what do add here and basically we want to display more data as the screen grows. Thus what changed:

- Move out date to separate column and add more padding for desktop
- Add padding to the chart on desktop, and move out sidebar layout to separate file
- Add sidebar to bills and budgets
- Add Eczar font

Closes https://github.com/material-components/material-components-flutter-gallery/issues/90
Closes https://github.com/material-components/material-components-flutter-gallery/issues/540

Note that this review is dependent on https://github.com/flutter/samples/pull/312 being merged.

Screenshots
=========
I verified that it looks good with large text scale as well as RTL but did not add those screenshots here.

Screen Shot 2020-02-05 at 14 15 42
Screen Shot 2020-02-05 at 14 15 48
Screen Shot 2020-02-05 at 14 15 55
Screen Shot 2020-02-05 at 14 16 02

### CODE REVIEW

File: gallery/lib/studies/rally/data.dart [hunk 1]
```diff
@@ -52,7 +58,8 @@ class AccountData {
 ///
 /// The [primaryAmount] is the amount due in USD.
 class BillData {
-  const BillData({this.name, this.primaryAmount, this.dueDate});
+  const BillData(
+      {this.name, this.primaryAmount, this.dueDate, this.isPaid = false});
```
Comment (clocksmith) [inline gallery/lib/studies/rally/data.dart]: trailing comma here?

File: gallery/lib/studies/rally/tabs/sidebar.dart [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        SizedBox(height: 8),
+        Text(
+          title,
+          style:
+              textTheme.body1.copyWith(fontSize: 16, color: RallyColors.gray60),
```
Comment (clocksmith) [inline gallery/lib/studies/rally/tabs/sidebar.dart]: trailing comma?

File: gallery/lib/studies/rally/tabs/sidebar.dart [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        ),
+        SizedBox(height: 8),
+        Text(
+          value,
+          style: textTheme.body2.copyWith(fontSize: 20),
+        ),
+        SizedBox(height: 8),
+        Container(color: RallyColors.primaryBackground, height: 1),
```
Comment (clocksmith) [inline gallery/lib/studies/rally/tabs/sidebar.dart]: trailing comma?

File: gallery/lib/studies/rally/data.dart (hunk 1 above)
Comment (perclasson) [reply gallery/lib/studies/rally/data.dart]: Done!

**Pull request merged**